src/test/java/com/checkmarx/flow/cucumber/integration/cli/sca/ScaCliSteps.java

```diff
@@ -24,64 +20,67 @@
 import java.io.IOException;
 import java.lang.reflect.InvocationTargetException;
 import java.util.Collections;
+import java.util.List;
 
+/**
+ * This step implementation relies on specific projects that already exist in SCA (see customScaProjectName).
+ */
 @SpringBootTest(classes = {CxFlowApplication.class, JiraTestUtils.class})
 @Slf4j
 @RequiredArgsConstructor
 public class ScaCliSteps {
     private static final String REPO_ARGS = " --repo-url=https://github.com/cxflowtestuser/public-rest-repo --repo-name=CLI-public-rest-repo --branch=master --blocksysexit";
-    private static final String GITHUB_REPO_ARGS = REPO_ARGS.intern() + " --github ";
+    private static final String GITHUB_REPO_ARGS = REPO_ARGS + " --github ";
```

Comment:
the intern prevents creation of extra string

Reply:
IntelliJ highlights `.intern()` and says that "Call to 'intern()' on compile-time constant is unnecessary"
So GITHUB_REPO_ARGS is evaluated during compilation.